string object_type
string camera_frame
geometry_msgs/Pose location
std_msgs/Header image_header
